### ISSUE TITLE
Consider update to Drupal Guide format

### ISSUE THREAD

**Issue by laurelthrash** [NCIOCPL/ncids#1083]

**Comment by andyvanavery31** [comment]
Continue to think through this ticket throughout R4, and plan to make updates after R4 (Article Pages) is complete.

**Comment by laurelthrash** [comment]
@welshja I agree this can transition to Jira. I suspect this made it to "Next Up" as we explore/prepare content for the NCI User Community Space with migrated MyNCI content. Cc: @andyvanavery31
